### PR TITLE
[DevOps] Fix missed variables format.

### PR DESCRIPTION
**Pull request by mandel-macaque**

### CODE REVIEW

**Review by dalexsoto — APPROVED** [review]

**Review by rolfbjarne — CHANGES_REQUESTED** [review]
I'm assuming this is trying to fix the broken package upload:

> Artifacts were not provided.

but it didn't...

**Review by tj-devel709 — APPROVED** [review]

File: tools/devops/automation/templates/main-stage.yml [hunk 1]
```diff
@@ -249,8 +255,8 @@ stages:
   - template: ./sign-and-notarized/funnel.yml
     parameters:
       isPR: ${{ parameters.isPR }}
-      azureStorage:  variables['azureStorage']
-      azureContainer: variables['azureContainer']
+      azureStorage:  ${{ paramenters.azureStorage }}
+      azureContainer: ${{ parameters..azureContainer }}
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/main-stage.yml:259]: Typo?

```suggestion
      azureContainer: ${{ parameters.azureContainer }}
```

File: tools/devops/automation/templates/main-stage.yml [hunk 2]
```diff
@@ -249,8 +255,8 @@ stages:
   - template: ./sign-and-notarized/funnel.yml
     parameters:
       isPR: ${{ parameters.isPR }}
-      azureStorage:  variables['azureStorage']
-      azureContainer: variables['azureContainer']
+      azureStorage:  ${{ paramenters.azureStorage }}
```
Comment (rolfbjarne) [inline tools/devops/automation/templates/main-stage.yml:258]: Typo?

```suggestion
      azureStorage:  ${{ parameters.azureStorage }}
```

File: tools/devops/automation/templates/main-stage.yml [hunk 3]
```diff
@@ -249,8 +255,8 @@ stages:
   - template: ./sign-and-notarized/funnel.yml
     parameters:
       isPR: ${{ parameters.isPR }}
-      azureStorage:  variables['azureStorage']
-      azureContainer: variables['azureContainer']
+      azureStorage:  ${{ variables.azureStorage }}
+      azureContainer: ${{ variables.azureContainer }}
```
Comment (mandel-macaque) [inline tools/devops/automation/templates/main-stage.yml:259]:
```suggestion
      azureStorage:  ${{ parameters.azureStorage }}
      azureContainer: ${{ parameters.azureContainer }}
```

**Review by rolfbjarne — APPROVED** [review]

**Pull request merged**